vnc_lite.html

```diff
@@ -167,25 +165,14 @@
         var password = WebUtil.getConfigVar('password', '');
         var path = WebUtil.getConfigVar('path', 'websockify');
 
-        // If a token variable is passed in, set the parameter in a cookie.
-        // This is used by nova-novncproxy.
-        var token = WebUtil.getConfigVar('token', null);
-        if (token) {
-            // if token is already present in the path we should use it
-            path = WebUtil.injectParamIfMissing(path, "token", token);
-
-            WebUtil.createCookie('token', token, 1)
-        }
-
         // Connect
         (function() {
 
             status("Connecting");
 
             // Build the websocket URL used to connect
             var url;
-            if (WebUtil.getConfigVar('encrypt',
-                                     (window.location.protocol === "https:"))) {
+            if (window.location.protocol === "https:") {
```

Comment:
I think this change means we can drop the port logic above (the one that sets 80/443).

Reply:
done